Scaffolding.sln

```diff
@@ -77,7 +77,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ClassLibrary1", "test\TestA
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.VisualStudio.Web.CodeGeneration.MSBuild.Test", "test\Microsoft.VisualStudio.Web.CodeGeneration.MSBuild.Test\Microsoft.VisualStudio.Web.CodeGeneration.MSBuild.Test.csproj", "{6C1398FF-A8C1-45A9-9796-6A60B2B011E1}"
```

Comment:
I did not shorten any of the tests because I didn't think they would affect the Coherence-Signed thing one way or another, but I can change them too for consistency.